src/main/java/core/basesyntax/figures/RightTriangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.figures;
+
+public class RightTriangle extends Figure {
+
+    private final double base;
+    private final double height;
+
+    public RightTriangle(String color, double base, double height) {
+        super(color);
+        this.base = base;
+        this.height = height;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.round((base * height / 2) * 10.0 / 10.0);
```

Comment:
There's no need to round the result of the area calculation using `Math.round` with additional multiplication and division by 10.0. If you want to limit the number of decimal places, consider using `String.format` or `DecimalFormat` when printing the value.